scripts/generate_dump

```diff
@@ -868,6 +872,9 @@ while getopts ":xnvhzs:" opt; do
         z)
             DO_COMPRESS=false
             ;;
+        a)
+            ALLOW_PROCESS_STOP="-a"
```

Comment:
Should this read `ALLOW_PROCESS_STOP="-f"`? Because this is the flag which gets passed to `/usr/bin/hw-management-generate-dump.sh` on line 767. I assume `/usr/bin/hw-management-generate-dump.sh` is still expecting `-f`.

Reply:
This is not yet finalized. we can change it to any letter at this point. Once we decide, we can ask vendors to follow.